README.md

```diff
@@ -1,111 +1,194 @@
+`cellxgene` is an interactive data explorer that leverages modern web development techniques to visualize large single-cell transcriptomics datasets, such as the data coming from the [Human Cell Atlas](https://humancellatlas.org). We hope to enable biologists and computational researchers to explore their data, and to demonstrate general, scalable, and reusable patterns for scientific data visualization.
 
-## Features
+<img src="./docs/cellxgene-demo-1.gif" width="200" height="200" hspace="30"><img src="./docs/cellxgene-demo-2.gif" width="200" height="200" hspace="30"><img src="./docs/cellxgene-demo-3.gif" width="200" height="200" hspace="30">
 
-- **Visualization at scale:** built with [WebGL](https://www.khronos.org/webgl/), [React](https://reactjs.org/) & [Redux](https://redux.js.org/) to handle visualization of at least 1 million cells.
+## features
 
-- **Interactive exploration:** select, cross-filter, and compare subsets of your data with performant indexing and data handling.
+- Built for **visualization at scale** using [WebGL](https://www.khronos.org/webgl/), [React](https://reactjs.org/), and [Redux](https://redux.js.org/) to handle at least 1 million cells.
+- Enables **interactive exploration** with performant indexing and data handling for selection, cross-filtering, and comparisons across subsets of data.
+- Support for **computational backends**  through a client-server model that can support a range of existing analysis packages for backend computational tasks (currently built for [scanpy](https://github.com/theislab/scanpy)).
 
-- **Flexible API:** the cellxgene client-server model is designed to support a range of existing analysis packages for backend computational tasks (eg scanpy), integrated with client-side visualization via a [REST API](https://restfulapi.net/).
+## getting started
 
-## Getting Started
+You'll need **python 3.6** and **Google Chrome**. The web UI is tested on OSX and Windows using Chrome, and the python CLI is tested on OSX and Ubuntu (via WSL/Windows). It should work on other platforms, but if you run into trouble let us know (see [help](#help-and-contact) below).
 
-**Requirements**
+To install run
+
+```
+pip install cellxgene
+```
+
+To start the tool and see the help information call
+
+```
+cellxgene --help
+```
+
+There are two primary subcommands -- `launch` and `prepare`
+
+- The `launch` command loads a dataset and starts the interactive explorer in your web browser
+- The optional `prepare` command takes an existing dataset in one of several formats and applies minimal preprocessing and reformatting so that `launch` can use it
+
+To explore a dataset call
+
+```
+cellxgene launch dataset.h5ad --open
+```
+
+If you want an example dataset download [this file](https://github.com/chanzuckerberg/cellxgene/raw/master/example-dataset/pbmc3k.h5ad) and then call
+
+```
+cellxgene launch pbmc3k.h5ad --open
+```
+You should see your web browser open with the following
```

Comment:
If on OSX, you should see...  

see comment below regarding --open

Reply:
added note about this